developer-docs-site/docs/reference/telemetry.md

```diff
@@ -3,32 +3,101 @@ title: "Telemetry"
 slug: "telemetry"
 ---
 
-At Aptos Labs, we develop software and services for the greater Aptos community and ecosystem. On top of community feedback, we use telemetry to help improve the decentralization of the network by understanding how our software is being deployed and run.
+When you operate a node on an Aptos network, your node can be set to send telemetry data to Aptos Labs. You can disable telemetry at any point. If telemetry remains enabled, Aptos node binary will send telemetry data in the background.
 
-The Aptos node binary collects telemetry such as software version, operating system information, and IP address. See [Types of information collected](#types-of-information-collected).
+The Aptos node binary running on your node collects telemetry data such as software version, operating system information and the IP address of your node. This telemetry data is used to enhance the decentralization of the network.
 
+:::tip No personal information is collected
 The Aptos node binary does **not** collect personal information such as usernames or email addresses.
+:::
 
-Users can disable telemetry at any point. If telemetry remains enabled, Aptos node binary will send telemetry data in the background.
+## Metrics collected
 
-# Disabling telemetry
+### Core metrics
```

Comment:
I am not sure we need to detail each of the events and their keys. It is mostly for Aptos' ability to understand/debug what is going on in the nodes. We could be adding and removing keys from it and could be causing unnecessary friction for something operators dont really action on. Can we instead just point to source? I also think we should remove the frequency and key counts.